### PR TITLE
Adds curl command option to download latest VMware image. Fixes sever…

### PR DESCRIPTION
**Pull request by Michael-Vincerra**
…al layout errors.

Signed-off-by: Michael Vincerra <michaelx.vincerra@intel.com>

### CODE REVIEW

**Comment by mltullis** [comment]
LGTM. After Bun approves, I'll merge.

File: source/clear-linux/get-started/virtual-machine-install/vmw-player-preconf.rst [hunk 1]
```diff
@@ -39,25 +39,25 @@ Install the VMware Workstation Player hypervisor
...
-      #. Start a terminal emulator.
+     #. Start a terminal emulator.
 
-      #. Start the installer by issuing the command below and follow the guided
-         steps.
+     #. Start the installer by issuing the command below and following the
+        guided steps.
 
-         .. code-block:: console
+        .. code-block:: console
```
Comment (bktan8) [inline source/clear-linux/get-started/virtual-machine-install/vmw-player-preconf.rst]: This should code-block should be "bash".

**Comment by bktan8** [comment]
@Michael-Vincerra - curiosity why there were "layout errors" if this was a published document?

File: source/clear-linux/get-started/virtual-machine-install/vmw-player-preconf.rst (hunk 1 above)
Comment (Michael-Vincerra) [reply source/clear-linux/get-started/virtual-machine-install/vmw-player-preconf.rst]: I changed this and one other instance from console to bash--given this is new plan of record.

Comment (Michael-Vincerra) [reply source/clear-linux/get-started/virtual-machine-install/vmw-player-preconf.rst]: Correction: I will leave the final instance of console as is, under "Enable UEFI boot support". Stet.

**Comment by Michael-Vincerra** [comment]
@bktan8  - I was curious about layout errors too. I just fixed them. 
FYI: There's a broken link on this page too, but I will try to resolve that separately.

**Pull request merged**